sign.js

```
@@ -0,0 +1,100 @@
+  var validation_result = sign_options_schema.validate(options);
+
+  if (validation_result.error) {
+    throw validation_result.error;
+  }
+
+  var timestamp = payload.iat || Math.floor(Date.now() / 1000);
+
+  if (!options.noTimestamp) {
+    payload.iat = timestamp;
+  } else {
+    delete payload.iat;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)